### ISSUE TITLE
iOS controls appear over plyr control when exiting native fullscreen by swiping down to exit

### ISSUE THREAD

**Issue by mrleblanc101** [sampotts/plyr#2278]
### Expected behaviour
iOS controls should never be shown

### Actual behaviour
iOS controls appear over plyr control when exiting native fullscreen by swiping down the player

### Steps to reproduce
- Confirgure Plyr to use iOS native fullscreen
- Start video playback
- Swiper down to close the video player (Don't press the close button in the top left)
- iOS controls will show over the Plyr control on the video preview

### Environment

- Browser: Safari
- Version: 14
- Operating System: iOS
- Version: 14.7

Example swiping down to exit
![2021-07-20 10 45 27](https://user-images.githubusercontent.com/17392251/126344814-b70c568d-1a40-48d8-ae7d-131c351ad4a8.gif)

Example using the close button
![2021-07-20 10 43 49](https://user-images.githubusercontent.com/17392251/126344803-80bce624-a95b-4492-beef-4c37bfc7c1ea.gif)

**Comment by mannyveloso** [comment]
Same issue here, iOS controls take over in fullscreen on iOS. Note this doesn't happen on iPadOS.

**Comment by mrleblanc101** [comment]
@mannyveloso I don't think your issue is related. My issue is after exiting Fullscreen by swiping down. Not while in fullscreen.

**Comment by mannyveloso** [comment]
So you see the plyr controls when playing on iOS ie: if you touch the video while playing you see the plyr controls, not the iOS controls? That's interesting, we see the exact opposite: when we play a video the iOS controls take over and when we swipe down we see the plyr controls.

What iOS are you on? Ours is on 14.7.1

**Comment by mrleblanc101** [comment]
@mannyveloso Did you look at the gif or...? When in fullscreen, iOS controls are shown because I have `fullscreen: { iosNative: true }`.

**Comment by mannyveloso** [comment]
I did; the 2nd video shows iOS video controls before the close button is clicked and what I presume are your custom controls when the video stops.

**Comment by mrleblanc101** [comment]
Can you create you own issue instead of polluting mine ? Clearly your problem is totally unrelated. The fact that iOS control display in fullscreen is totally normal as I said because I have the option `{ fullscreen: iosNative: true }`. Maybe you should go read the documentation: ![image](https://user-images.githubusercontent.com/17392251/128254003-d94f9906-600a-45ac-854b-c574b6d54a93.png)

**Comment by mannyveloso** [comment]
Yes I will to be stopping polluting your issue. Thanks for your help and understanding, I'm sure that I'll try and do the same sometime in the future.
